modules/login/php/module.class.inc

```diff
@@ -47,4 +49,22 @@ class Module extends \Module
     {
         return "Login";
     }
+
+    /**
+     * Check whether a user is logined or not, if yes, redirect to the base URL
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request
+     *
+     * @return ResponseInterface the outgoing PSR7 response
+     */
+    public function handle(ServerRequestInterface $request) : ResponseInterface
+    {
+        $user = \User::singleton();
+        if (!($user instanceof \LORIS\AnonymousUser)) {
+            return (new \LORIS\Http\Response())
+                ->withStatus(307)
+                ->withHeader('Location', '/');
```

Comment:
```suggestion
                ->withHeader('Location', \NDB_Factory::singleton()->settings()->getBaseURL());
```

This (untested) will work regardless of whether LORIS is hosted under the base of the domain name.